### ISSUE TITLE
`pip install csv2table` fails

### ISSUE THREAD

**Issue by K4zuki** [ickc/csv2table#1]
### python2
```
$ pip install csv2table
Collecting csv2table
  Using cached csv2table-0.4.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-hkPMOj/csv2table/setup.py", line 23, in <module>
        with open("csv2table.py") as f:
      File "/usr/lib/python2.7/codecs.py", line 896, in open
        file = __builtin__.open(filename, mode, buffering)
    IOError: [Errno 2] No such file or directory: 'csv2table.py'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-hkPMOj/csv2table/
```

### python3
```
$ pip3 install csv2table
Collecting csv2table
  Using cached csv2table-0.4.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-sf8msica/csv2table/setup.py", line 23, in <module>
        with open("csv2table.py") as f:
      File "/usr/lib/python3.5/codecs.py", line 895, in open
        file = builtins.open(filename, mode, buffering)
    FileNotFoundError: [Errno 2] No such file or directory: 'csv2table.py'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-sf8msica/csv2table/
```
---
looks like simple path problem; can you fix, or is it my problem?

**Comment by ickc** [comment]
@K4zuki, sorry, this got buried and I didn't read this.

Which OS were you using?

**Comment by K4zuki** [comment]
This is **Ubuntu 16.04LTS**.
As I am not near the machine right now, I will report back to you after confirmation

**Comment by ickc** [comment]
@K4zuki, it should be fixed right now in v0.4.2. Let me know if it isn't.

**Comment by K4zuki** [comment]
Confirmed fixed on Windows7/OSX/Ubuntu for Py2/Py3 by 0.4.2
Thanks for your help!
